### ISSUE TITLE
"Out of Date" Error in Unorganized Text

### ISSUE THREAD

**Issue by alexvonme** [ChrisChinchilla/vale-vscode#46]
I get a ``The suggestion was not applied because it is out of date`` error from the extension when I apply fixes through actions in unorganized test files. By unorganized, I refer to text files where there aren't clear sentence separations with punctuation. I've noticed this issue in some of the mini test files I began writing and replicated the error once more with this [test document](https://github.com/openSUSE/suse-vale-styleguide/pull/103#discussion_r1649396781).

The problem isn't a critical one. I realized that it probably just prompted a save before I could continue editing. However, it often required multiple saves in succession (and maybe waiting?) before I could continue without encountering the error. I also had to try editing around the document to be able to edit the line I wanted, like editing the first and 3 paragraphs before being able to edit the second paragraphs 3rd line.

**Comment by ChrisChinchilla** [comment]
@alexvonme Yeah, this is a long-term limitation of the extension and the way VSCode works. I personally don't see the error that much anymore, so I am not sure if it may be fully resolved now anyway. Have you seen it recently?
